tools/osbuilder/rootfs-builder/init_trusted_storage.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+[ -n "${BASH_VERSION:-}" ] && set -o errtrace
+[ -n "${DEBUG:-}" ] && set -o xtrace
+
+device_num=${1:-}
+if [ -z "$device_num" ]; then
+        echo "ERROR: invalid arguments" >&2
+        exit 1
+fi
+
+enable_data_integrity="true"
+if [ -n "${2-}" ]; then
```

Comment:
What is the purpose of `"${2-}"` here (not sure why you are not using `${2:-}` or even just `$2`)?

Reply:
That's to stop the shell immediately exiting of `$2` is *not* set.